CRM/Activity/Form/Search.php

```diff
@@ -65,6 +65,7 @@ public function getDefaultEntity(): string {
    * @throws \CRM_Core_Exception
    */
   public function preProcess(): void {
+    throw new \Exception('aha!');
```

Comment:
Have you considered the right exception here might be `CRM_Core_Exception()` - any total site breakage should be Civi-specific IMHO